cmd/copy.go

```diff
@@ -1128,6 +1140,9 @@ func init() {
 	cpCmd.PersistentFlags().StringVar(&raw.metadata, "metadata", "", "upload to Azure Storage with these key-value pairs as metadata.")
 	cpCmd.PersistentFlags().StringVar(&raw.contentType, "content-type", "", "specifies content type of the file. Implies no-guess-mime-type.")
 	cpCmd.PersistentFlags().StringVar(&raw.contentEncoding, "content-encoding", "", "upload to Azure Storage using this content encoding.")
+	cpCmd.PersistentFlags().StringVar(&raw.contentDisposition, "content-disposition", "", "upload to Azure Storage using this content disposition.")
```

Comment:
These headers are returned when downloading. Please update these messages to be more clear.

Refer to: https://docs.microsoft.com/en-us/rest/api/storageservices/put-blob